amcl/src/amcl/map/map_store.c

```diff
@@ -74,6 +75,19 @@ int map_load_occ(map_t *map, const char *filename, double scale, int negate)
     return -1;
   }
 
+  if ( (long)width * (long)height > INT_MAX ){
```

Comment:
What does comparing with INT_MAX mean in this function?

Reply:
I fixed it.

Reply:
OK.